examples/relayer-sei-example/example.env

```
@@ -0,0 +1,4 @@
+REDIS_URL=redis://localhost:6379
+API_KEY=8aa82468-3c3d-47db-aa05-7ef176d78417
+WEBHOOK_SIGNING_KEY=8aa82468-3c3d-47db-aa05-7ef176d78417
+KEYSTORE_PASSPHRASE=_Abcdefg123456
```

Leave these fields empty
```suggestion
REDIS_URL=redis://localhost:6379
API_KEY=
WEBHOOK_SIGNING_KEY=
KEYSTORE_PASSPHRASE=
```

---

the idea of the example is to quickly run the api and tests and have them succeed, this is not a real config that anyone would use. And I also renamed the env to example.env so I don't see the issue